examples/decoding/plot_receptive_field.py

```diff
@@ -0,0 +1,124 @@
+#
+# License: BSD (3-clause)
+
+
+from mne.decoding import ReceptiveField
+from sklearn.linear_model import Ridge
+from sklearn.model_selection import KFold
+from sklearn.preprocessing import scale
+from scipy.io import loadmat
+
+import numpy as np
+import matplotlib.pyplot as plt
+import mne
+
+
+###############################################################################
+# Load the data from the publication
+path = mne.datasets.mtrf.data_path()
+n_decim = 2  # To speed up computation
+data = loadmat(path + '/speech_data.mat')
+raw = data['EEG'].T
+speech = data['envelope'].T
+sfreq = float(data['Fs'].squeeze())
+sfreq /= n_decim
+speech = mne.filter.resample(speech, 1, n_decim, 'auto')
+raw = mne.filter.resample(raw, 1, n_decim, 'auto')
+
+
+# Read in channel positions and create our MNE objects from the raw data
+mon = mne.channels.read_montage('biosemi128')
+mon.selection = mon.selection[:128]
+layout = mne.channels.read_layout('Vectorview-all')
+info = mne.create_info(mon.ch_names[:128], sfreq, 'eeg', montage=mon)
+layout = mne.channels.make_eeg_layout(info)
+raw = mne.io.RawArray(raw, info)
+
+###############################################################################
+# Plot a sample of brain and stimulus activity
+fig, ax = plt.subplots()
+ax.plot(scale(raw[:][0][..., :800].T), color='k', alpha=.1)
+ax.plot(scale(speech[0, :800]), color='r', lw=2)
+ax.set(title="Sample brain and stimulus activity", xlabel="Time (s)")
+mne.viz.tight_layout()
+
+###############################################################################
+# Create and fit a receptive field model
+
+# Define the delays that we'll use in the receptive field
+tmin, tmax = -.4, .2
+
+# Initialize the model
+spec_names = ['freq_%s' % ii for ii in range(speech.shape[0])]
+mod = Ridge()
+rf = ReceptiveField(tmin, tmax, sfreq, spec_names)
+delays_time = rf.delays / float(sfreq)
+
+n_cv = 3
+kf = KFold(n_cv)
+
+# Iterate through folds, fit the model, and predict/test on held-out data
+coefs = np.zeros([n_cv, len(raw.ch_names), len(delays_time)])
+scores = np.zeros([n_cv, len(raw.ch_names)])
+for ii, (tr, tt) in enumerate(kf.split(speech.T)):
+    print('CV iteration %s' % ii)
+    for jj, i_ch in enumerate(raw[:][0]):
+        rf.fit(speech[..., tr], i_ch[tr])
+        preds = rf.predict(speech[..., tt]).squeeze()
+        scores[ii, jj] = np.corrcoef(preds, i_ch[tt][rf.mask_pred])[1, 0]
+        coefs[ii, jj] = rf.coef_
+mn_coefs = coefs.mean(0)
+mn_scores = scores.mean(0)
+
+# Plot mean prediction scores across all channels
+fig, ax = plt.subplots()
+ix_chs = range(len(raw.ch_names))
+ax.plot(ix_chs, mn_scores)
+ax.axhline(0, ls='--', color='r')
+ax.set(title="Mean prediction score", xlabel="Channel", ylabel="Score ($r$)")
+mne.viz.tight_layout()
+
+###############################################################################
+# Investigate model coefficients
+#
+# Here we'll look at how coefficients are distributed across time lag as well
```

Comment:
how coefficients => how the linear coefficients, sometimes referred to as beta values, 

lag => lags